evap/staff/templates/staff_evaluation_textanswers_full.html

```diff
@@ -37,7 +37,11 @@
 {% block additional_javascript %}
     <script type="text/javascript">
         document
-            .querySelectorAll(".full-update-textanswer-form input[type='radio']")
-            .forEach(radio => radio.addEventListener("change", () => radio.form.submit()));
+            .querySelectorAll(".full-update-textanswer-form input[type='radio'], .full-textanswer-flag-form input[type='radio']")
+            .forEach(input => input.addEventListener("change", () => {
+                input.form.submit();
+                input.disabled = true;
+                setTimeout(() => input.disabled = false, 500);
```

Comment:
Out of curiosity: What happens here on a network error? I'd like it if the input button only was re-enabled once we knew that the server processed our request, but I guess that's a bit hard to do here?

Reply:
yes we could, but that would require submitting through javascript `fetch` again I think

Reply:
So what happens currently? Is the error simply ignored.

Reply:
Browser should just timeout and display a sad face, as if one would just submit a form and it never responds

Reply:
If the network is just slow, it doesn't matter if the form is submitted again, because it will just write the same value again

Reply:
But I can agree that it is a bit icky

Reply:
Alright, works for me